jwst/datamodels/schemas/dark.schema.yaml

```diff
@@ -1,25 +1,30 @@
 title: Dark current data model
 allOf:
 - $ref: referencefile.schema.yaml
+- $ref: instrument_name.schema.yaml
+- $ref: detector.schema.yaml
+- $ref: filter_core.schema.yaml
```

Comment:
Is it going to be confusing for developers to have to learn which individual keyword schema files need to use the _core suffix in their names? And does this mean it's an identical copy of what's in the science core.schema, as opposed to the way it's defined for use in reference files?